src/commands/codepush/lib/react-native-utils.ts

```diff
@@ -455,7 +455,7 @@ export async function runHermesEmitBinaryCommand(
   });
 }
 
-function parseBuildGradleFile(gradleFile: string) {
+function getBuildGradleFilePath(gradleFile: string) {
```

Comment:
let's rename these two methods to `getGradleBuildFilePath` and `parseGradleBuildFile`, for the sake of consistency with `parseGradlePropertiesFile`.

Also I would put `hermes` somehow in this names of the parsing methods, because this file is `react-native-utils` and function names such as `parseGradleBuildFile` don't reflect the logic inside. It should be more specific to make it clear. F.e. `checkHermesInGradlePropertiesFile` and `checkHermesInGradleBuildFile`.

Reply:
@BANG88 Could you please rename the name of this function in the way Dmitry described?

Reply:
Okay. but  rename which one ? thanks.

Reply:
Firstly, you have an error in the names of the functions, you have two functions, one of which is responsible for getting the path to the gradle build file and the second for parsing this file, but nevertheless they are both called the same `getBuildGradleFilePath`. You need to rename the function that is responsible for getting the path to the gradle build file to `getGradleBuildFilePath`. And the one that is responsible for parsing on `checkHermesInGradleBuildFile`. Also rename the function that is responsible for parsing the gradle property file to `checkHermesInGradlePropertiesFile`, because the parsing functions refer specifically to the logic of finding the attribute of the included Hermes engine.

Reply:
I'm sorry, I really am too stupid, blind.